dom/nodes/Document-createEvent.html

```diff
@@ -99,48 +101,57 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
-  "MIDIMessageEvent",
+  "KeyEvents",
   "MediaEncryptedEvent",
   "MediaKeyEvent",
   "MediaKeyMessageEvent",
   "MediaQueryListEvent",
   "MediaStreamEvent",
   "MediaStreamTrackEvent",
+  "MIDIConnectionEvent",
+  "MIDIMessageEvent",
+  "MouseScrollEvents",
   "MutationEvent",
   "NotificationEvent",
+  "NotifyPaintEvent",
   "OfflineAudioCompletionEvent",
   "OrientationEvent",
+  "PageTransition", // Yes, with no "Event"
   "PageTransitionEvent",
   "PointerEvent",
   "PopStateEvent",
+  "PopUpEvents",
   "PresentationConnectionAvailableEvent",
   "PresentationConnectionCloseEvent",
   "ProgressEvent",
   "PromiseRejectionEvent",
   "PushEvent",
-  "RTCDTMFToneChangeEvent",
-  "RTCDataChannelEvent",
-  "RTCIceCandidateEvent",
   "RelatedEvent",
   "ResourceProgressEvent",
-  "SVGEvent",
-  "SVGZoomEvent",
+  "RTCDataChannelEvent",
+  "RTCDTMFToneChangeEvent",
+  "RTCIceCandidateEvent",
+  "ScrollAreaEvent",
   "SecurityPolicyViolationEvent",
   "ServicePortConnectEvent",
   "ServiceWorkerMessageEvent",
+  "SimpleGestureEvent",
   "SpeechRecognitionError",
   "SpeechRecognitionEvent",
   "SpeechSynthesisEvent",
   "StorageEvent",
+  "SVGEvent",
+  "SVGZoomEvent",
   "SyncEvent",
   "TextEvent",
+  "TimeEvent",
   "TrackEvent",
   "TransitionEvent",
   "WebGLContextEvent",
   "WebKitAnimationEvent",
   "WebKitTransitionEvent",
-  "WheelEvent"
+  "WheelEvent",
+  "XULCommandEvent",
```

Comment:
I always put a trailing comma in lists, so that if someone adds an entry at the end, a) it won't break if they forget to add a comma, b) the diff won't show the last line as changed because of the added comma.  In fact, when writing this patch it initially broke because I added XULCommandEvent without adding the trailing comma!